<?xml version="1.0"?>
<launch>
    <node pkg="coverage_exploration" type="coverage_exploration_node" name="coverage_exploration_node" output="screen" >
        <param name="recieve_global_map_topic_name" value="/global_map"/>
        <param name="local_map_frame_name" value="base_link"/>
        <param name="global_map_frame_name" value="/odom"/>
        <param name="send_cover_map_topic_name" value="/cover_map"/>

        <param name="vehicle_length" value="4.9"/>
        <param name="vehicle_width" value="1.95"/>
        <param name="base2back" value="1.0"/>
        <param name="polygon_height" value="20"/>
        <param name="polygon_base_length" value="16.8"/>
        <param name="base2camera_length" value="1.5"/>
        <param name="left_base2camera_length" value="0.9"/>
        <param name="obs_threshold_" value="80"/>
        <param name="border_thickness" value="2"/>

     </node>

    <!--
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find astar_planner)/rviz/astar_planner.rviz"/>
    -->
</launch>